src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,107 @@
+.header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 24px 370px;
+
```

Comment:
No empty line is needed anywhere in this file.

Reply:
I have checked several times and tried to eliminate all unnecessary empty line :)